cmd/chaos-daemon/main.go

```diff
@@ -25,18 +26,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
-var log = ctrl.Log.WithName("chaos-daemon")
-
 var (
-	printVersion     bool
-	rawPort          string
-	containerRuntime string
+	log  = ctrl.Log.WithName("chaos-daemon")
+	conf = &chaosdaemon.Config{Host: "0.0.0.0"}
+
+	printVersion bool
 )
 
 func init() {
 	flag.BoolVar(&printVersion, "version", false, "print version information and exit")
-	flag.StringVar(&rawPort, "port", "", "the port which server listens on")
-	flag.StringVar(&containerRuntime, "runtime", "docker", "current container runtime")
+	flag.IntVar(&conf.GRPCPort, "grpc-port", 8080, "the port which grpc server listens on")
```

Comment:
this two port is too common, did we need to change @cwen0 @YangKeao

Reply:
I agree to change the default port

Reply:
Yes, but change to what? Any recommendations? But since we can set a large port (3xxxx) in the helm chart so I think it is ok to not change it.

Reply:
Maybe use the same value with values.yaml?

Reply:
That sounds good. I will set to that value.

Reply:
Actually we don't need to set http port, it is ok to be a common port since we don't expose it as a host port